services/comments.js

```diff
@@ -110,6 +110,7 @@ module.exports = class CommentsService {
       $set: {
         body,
         status,
+        metadata,
```

Comment:
Here you are essentially removing all other metadata. See https://github.com/coralproject/talk/blob/master/services/settings.js#L59 on how to solve it.